test/fixtures/pods-acceptance/output.js

```diff
@@ -0,0 +1,134 @@
+module.exports = {
```

Comment:
You could require that and return it if you wanted (just to ensure both stay in sync)...